userbenchmark/ddp_experiments/__init__.py

```diff
@@ -557,7 +581,7 @@ def get_hf_T5_large_config(nodes, model_args):
         return ExperimentParams(config, args_copy, copied_model_args, is_reference=fsdp_is_reference(backend_name))
 
     def get_hf_GPT2_large_config(nodes, model_args):
```

Comment:
we discussed before,  i feel like it'd be cleaner to just have one helper that takes model, wrap_fn, and any other args (none currently, right?) that are model specific rather than having many separate functions?